lib/selendroid.js

```diff
@@ -51,12 +54,25 @@ class SelendroidServer {
   }
 
   async buildNewModServer () {
-    logger.info(`Repackaging selendroid for '${this.appPackage}'`);
+    logger.info(`Repackaging selendroid for: '${this.appPackage}'`);
     let packageTmpDir = path.resolve(this.tmpDir, this.appPackage);
     let newManifestPath = path.resolve(this.tmpDir, 'AndroidManifest.xml');
-    logger.info(`Creating new manifest`);
-    await fs.mkdir(packageTmpDir);
-    await fs.copyFile(SE_MANIFEST_PATH, newManifestPath);
+    logger.info(`Creating new manifest: '${newManifestPath}'`);
+
+    logger.info(`MANIFEST PATH: ${SE_MANIFEST_PATH}`);
+    logger.info(await fs.exists(SE_MANIFEST_PATH));
```

Comment:
Yes. This is me messing about. The CI system is failing to get all the files in order.

But this logging will be gone once the issue is figured out. :)